src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/cluster.module.ts

```diff
@@ -102,7 +104,10 @@ import { MultiClusterDetailsComponent } from './multi-cluster/multi-cluster-deta
     CheckboxModule,
     GridModule,
     ProgressIndicatorModule,
-    ButtonModule
+    ButtonModule,
+    InputModule,
+    ButtonModule,
```

Comment:
This is repeated, right?
```suggestion
    ButtonModule,
    InputModule,
```